From 8cb1187a0f684f2031d9cae7996efbb3fc4fa590 Mon Sep 17 00:00:00 2001
From: Stella Laurenzo <stellaraccident@gmail.com>
Date: Tue, 11 Feb 2025 22:10:22 -0800
Subject: [PATCH] Use idomatic approach to extending CMAKE_MODULE_PATH.

There are many reasons why there may already be a CMAKE_MODULE_PATH defined. The idiomatic way to extend it in a project is via `list(APPEND`.
---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index df6755a..ecdcf77 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,9 +50,7 @@ endif()
 # Default to ON
 option(BUILD_SHARED_LIBS "Build using shared libraries" ON)
 ## Set default module path if not already set
-if(NOT DEFINED CMAKE_MODULE_PATH)
-    set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
-endif()
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 ## Include common cmake modules
 include(utils)
 
-- 
2.43.0

